docs/configuration/installation.rst

```diff
@@ -3,45 +3,9 @@
 Installation
 ############
 
-*Kinto* is meant to be relatively straightforward to install, and we've
-provided fairly extensive documentation to get you up and running. That said,
-some dependencies might make the process harder than it should be -
-we've got some notes on crypto libraries and PostgreSQL at the end of this page.
+Depending on the platform, and chosen configuration some libraries or
+extra services may become mandatory.
```

Comment:
Also, I'dd probably add a note about the fact that a public Moz demo instance is deployed, so the reader which would actually just want to quickly toy around with the API doesn't mess with the installation procedure.